app/sweepers/tag_sweeper.rb

```diff
@@ -1,6 +1,6 @@
 class TagSweeper < ActionController::Caching::Sweeper
```

Comment:
We are not that I know of. I left the file in place as I wanted to make sure others weighed in on moving away from Sweepers slowly.